examples/powertools-examples-core/cdk/README.md

```diff
@@ -0,0 +1,64 @@
+#  Powertools for AWS Lambda (Java) - Core Utilities Example
+
+This project demonstrates the Lambda for Powertools Java module - including 
+[logging](https://docs.powertools.aws.dev/lambda/java/core/logging/),
+[tracing](https://docs.powertools.aws.dev/lambda/java/core/tracing/), and
+[metrics](https://docs.powertools.aws.dev/lambda/java/core/metrics/).
+
+It is made up of the following:
+
+- [App.java](src/main/java/helloworld/App.java) - Code for the application's Lambda function.
+- [events](events) - Invocation events that you can use to invoke the function.
+- [AppTests.java](src/test/java/helloworld/AppTest.java) - Unit tests for the application code. 
+
+## Deploy the sample application
+
+This is a blank project for CDK development with Java.
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app.
+
+It is a [Maven](https://maven.apache.org/) based project, so you can open this project with any Maven compatible Java IDE to build and run tests.
+
+The minimum to deploy the app should be
+```bash 
+cdk bootstrap && cdk deploy
+```
+
+## Useful commands
+
+* `mvn package`     compile and run tests
+* `cdk ls`          list all stacks in the app
+* `cdk synth`       emits the synthesized CloudFormation template
+* `cdk deploy`      deploy this stack to your default AWS account/region
+* `cdk diff`        compare deployed stack with current state
+* `cdk docs`        open CDK documentation
+
+
+## Test the application
+
+Once the app is deployed, you can invoke the endpoint like this:
+
+```bash
+ curl https://[REST-API-ID].execute-api.[REGION].amazonaws.com/Prod/hello/
+```
+
+The response itself isn't particularly interesting - you will get back some information about your IP address.  If 
+you go to the Lambda Console and locate the lambda you have deployed, then click the "Monitoring" tab you will
+be able to find:
+
+* **View X-Ray traces** -  Display the traces captured by the traces module. These include subsegments for the
+different function calls within the example
+* **View Cloudwatch logs** - Display the structured logging output of the example
+
+Likewise, from the CloudWatch dashboard, under **Metrics**, **all metrics**,  you will find the namespaces `Another`
+and `ServerlessAirline`. The values in each of these are published by the code in
+[App.java](src/main/java/helloworld/App.java). 
+
+You can also watch the trace information or log information using the SAM CLI:
+```bash
```

Comment:
Does the SAM stuff belong here?

Reply:
not sure we should mention SAM here... even if we can use both together

Reply:
I rewrote this file completely now.